admin/query/src/main/java/org/codice/ditto/replication/admin/query/replications/persist/DeleteReplication.java

```diff
@@ -50,6 +51,18 @@ public BooleanField performFunction() {
     return successful;
   }
 
+  @Override
+  public void validate() {
+    super.validate();
+    if (containsErrorMsgs()) {
+      return;
+    }
+
+    if (!replicationUtils.configExists(id.getValue())) {
+      addErrorMessage(ReplicationMessages.configDoesNotExist());
+    }
+  }
```

Comment:
replicationUtils.deleteConfig(id) could be changed to just let the NotFoundException bubble up. Then in the performFunction() method you could just return null with the error message if you catch that exception. At least I think you can do that with the graphql endpoint.